lifecycle/lifecycle/django/registry/admin.py

```diff
@@ -30,17 +30,17 @@ class DeploymentAdmin(admin.ModelAdmin):
 
 
 class PublicEndpointRequestAdmin(admin.ModelAdmin):
-    list_display = ('get_fatman_name', 'get_fatman_version', 'endpoint', 'active')
+    list_display = ('get_job_name', 'get_job_version', 'endpoint', 'active')
     search_fields = ['endpoint']
     list_filter = ['fatman__name', 'endpoint', 'active']
 
-    @admin.display(ordering='fatman__name', description='Fatman name')
-    def get_fatman_name(self, obj):
-        return obj.fatman.name
+    @admin.display(ordering='fatman__name', description='Job name')
+    def get_job_name(self, obj):
+        return obj.job.name
```

Comment:
This is also part of the Django's model management, so I guess it's better to keep this whole file in the original form for now.